tf_agents/environments/random_tf_environment.py

```diff
@@ -88,7 +88,7 @@ def _sample_obs_and_reward(self):
   def _reset(self):
     """Resets the environment and returns the current time_step."""
     obs, _ = self._sample_obs_and_reward()
-    time_step = ts.restart(obs, self._batch_size)
+    time_step = ts.restart(obs, self._batch_size, reward_spec = self._time_step_spec.reward)
```

Comment:
nit: can you remove the spaces around the equals sign?

Reply:
Updated code in my version - hopefully this flows through to the PR